packages/cfd/src/features/Containers/cfd-password-modal/cfd-password-modal.tsx

```diff
@@ -410,17 +410,16 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
             return (
                 <React.Fragment>
                     <Localize
-                        i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{platform}}</0> <1>{{type}} {{jurisdiction_selected_shortcode}}</1> account. '
+                        i18n_default_text='Congratulations, you have successfully created your <0/>{{category}} {{platform}} {{type}} {{jurisdiction_selected_shortcode}} account. '
```

Comment:
@hirad-deriv Please check the type for deriv X , it still shows the `CFDs`
<img width="457" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/e7bc8601-4c47-425c-91d4-08189ce6eba8">